wordpress-mobile/WordPress-iOS
# [Insights Management] Add ability to organize cards

Fixes #12247 

This adds the ability to:
- Move a stat card up.
- Move a stat card down.
- Remove stat card from Insights.

To test:
- Go to Stats > Insights.
- Verify the ellipsis button appears in all headers.

<img width="400" alt="manage_insight_button" src="https://user-images.githubusercontent.com/1816888/66430052-8e3ac480-e9d6-11e9-8a44-23d1acda7888.png">

- Tap the button.
- Select 'Move up'.
  - Verify the card moves up one slot.
  - If the card is at the top (or just below the Customize card), 'Move up' is not an option.
- Select 'Move down'.
  - Verify the card moves down one slot.
  - If the card is at the bottom, 'Move down' is not an option.
- Select 'Remove from insights'.
  - Verify the stat card is removed.
   - Select 'Add new stats card'.
  - Verify the newly removed stat is available to add.
- Please test on iPhone and iPad.

---
NOTE: A constraint warning appears when the action sheet is displayed. I tried other actions sheets in the app, and they do the same. It appears to be an issue with the action sheet itself.

---
Update release notes:
